types/semantic-release/v17/OTHER_FILES.txt

```diff
@@ -0,0 +1 @@
+lib/get-git-auth-url.d.ts
```

Comment:
can you add a test for this CJS version? this would remove requirement to add OTHER_FILES.txt (non-standard solution)

Reply:
Is there documentation or an example on this? Going by the README I assumed OTHER_FILES.txt is the correct way for providing definitions for modules not exposed by the main file.